### PR TITLE
Fix KeySort and add test vector

### LINKED ISSUE #58: KeySort test vectors
I think it would be helpful for those applications that do not have built-in canonical key ordering to have reference test vectors for the lexicographic key sorting. My current operating assumption is that it's based on the 33-byte-serialization.

### PR DESCRIPTION
**Pull request by jonasnick**
Fixes #58

### CODE REVIEW

**Review by real-or-random — APPROVED** [review]
ACK 52c9e28d2cc011f244e149999a49d201bd64abdc

**Comment by jonasnick** [comment]
Added change log entry

**Review by real-or-random — APPROVED** [review]
ACK 04bac8eef65ee8cfce27f0b01667eeefdb4cd600

File: bip-musig2.mediawiki [hunk 1]
```diff
@@ -651,6 +651,7 @@ An exception to this rule is <code>MAJOR</code> version zero (0.y.z) which is fo
 The <code>MINOR</code> version is incremented whenever the inputs or the output of an algorithm changes in a backward-compatible way or new backward-compatible functionality is added.
 The <code>PATCH</code> version is incremented for other changes that are noteworthy (bug fixes, test vectors, important clarifications, etc.).
 
+* '''0.8.2''' (2022-08-30): Add ''KeySort'' test vector
```
Comment (robot-dreams) [inline bip-musig2.mediawiki]: Nit: Is it worth mentioning this?

```suggestion
* '''0.8.2''' (2022-08-30): Fix ''KeySort'' input length and add test vectors
```

**Review by robot-dreams — COMMENTED** [review]
ACK 04bac8eef65ee8cfce27f0b01667eeefdb4cd600

**Pull request merged**